app/views/internal/path_redirects/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="row m-3">
+  <div class="col">
+    <a href="<%= new_internal_path_redirect_path %>" class="btn btn-primary" role="button">Create a New Path Redirect</a>
+  </div>
+  <div class="col">
+    <%= search_form_for @q, url: internal_path_redirects_path, class: "form-inline justify-content-end" do |f| %>
+
+      <%= f.label :old_path_or_new_path_cont, "Old or new path", class: "sr-only" %>
+      <%= f.search_field :old_path_or_new_path_cont, placeholder: "Old or new path", class: "form-control mx-3" %>
+
+      <%= f.submit "Search", class: "btn btn-secondary" %>
+    <% end %>
+  </div>
+</div>
+
+<%= paginate @path_redirects %>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">ID</th>
+      <th scope="col">Old Path</th>
+      <th scope="col">New Path</th>
+      <th scope="col">Source</th>
+      <th scope="col">Version</th>
+      <th scope="col">
+        <%= content_tag :span, sort_link(@q, :created_at, "Date"), class: "mx-2" %>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @path_redirects.each do |path_redirect| %>
+      <% badge_type =
+           if path_redirect.version <= 3
+             "success"
+           elsif path_redirect.version <= 5
+             "warning"
+           else
+             "danger"
+           end %>
+
```

Comment:
Do we need this whitespace here? 👀

Reply:
Good catch! Technically, we don't. It doesn't show up in the markup and my thinking was it's a little more readable since the lines above it are using indentation heavily. It's harder to catch the closing `%>` on line 40 when it's up against the next block of code - especially since the opening `<%` is several lines above that.